systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -106,6 +104,7 @@
     String KAFKA_EXPORTER_DEPLOYMENT = "KafkaWithExporter";
     String KAFKA_CRUISE_CONTROL_DEPLOYMENT = "KafkaWithCruiseControl";
     String STATEFUL_SET = "StatefulSet";
+    String POD = "Pod";
```

Comment:
Why these all variables are not `final` Strings? Are they going to change?

Reply:
(If I'm right) this is in `public interface class` and if you add `final` it's just redundant.

Reply:
You cannot modify them, that's what `interface` do. You don't need to write `final ...` everywhere when you want a constants class -> so you do it this way. Is this right @Frawless?

Reply:
Cool, I did not know that. Just googled it. Thanks!

Reply:
It is because Java Interfaces are static classes and OFC cannot be instantiated. `Static` because at the runtime the value of the variable must be assigned in a context where instance does not exist. `Final`, because the value assigned to the interface variable is a constant that cannot be re-assigned by program code.

Reply:
Thanks @see-quick, as always you described it better.